ui-admin/src/App.css

```diff
@@ -18,6 +18,13 @@
   font-style: italic;
 }
 
+.btn-primary[aria-disabled="true"] {
```

Comment:
For buttons, Bootstrap has a "disabled" CSS class that will apply Bootstrap's default styles for a disabled button. I'd prefer to use it rather than have multiple different styles for disabled buttons.